lib/usd/translators/lightReader.cpp

```diff
@@ -42,20 +48,21 @@ PXRUSDMAYA_DEFINE_READER(UsdLuxDiskLight, args, context)
     return false;
 }
 
-PXRUSDMAYA_DEFINE_READER(UsdLuxDistantLight, args, context)
+PXRUSDMAYA_DEFINE_READER(UsdLuxDomeLight, args, context)
```

Comment:
Are the definitions swapped? it seems that usdluxdome light was after distant light and now it is before it. The diff is getting confused.

Reply:
It is just git diff getting confused.. I moved the dome light up so they can all be commented out together and that's it